--- wxPython-4.0.6.orig/wscript	2019-05-21 20:12:12.000000000 -0300
+++ wxPython-4.0.6/wscript	2019-05-22 18:42:57.255117941 -0300
@@ -316,12 +316,11 @@
         conf.env.CFLAGS_WXPY.append('-DSIP_MODULE_BASENAME=siplib')
         conf.env.CXXFLAGS_WXPY.append('-DSIP_MODULE_BASENAME=siplib')
 
-        # Add basic debug info for all builds
-        conf.env.CFLAGS_WXPY.append('-g')
-        conf.env.CXXFLAGS_WXPY.append('-g')
 
         # And if --debug is set turn on more detailed debug info and turn off optimization
         if conf.env.debug:
+            conf.env.CFLAGS_WXPY.append('-g')
+            conf.env.CXXFLAGS_WXPY.append('-g')
             for flags in [conf.env.CFLAGS_PYEXT, conf.env.CXXFLAGS_PYEXT]:
                 flags.extend(['-ggdb', '-O0'])
 
