diff --git a/wx/__init__.py b/wx/__init__.py
index 649af3a7..06cbefee 100644
--- a/wx/__init__.py
+++ b/wx/__init__.py
@@ -7,6 +7,15 @@
 # License:     wxWindows License
 #---------------------------------------------------------------------------
 
+import os
+import platform
+
+# Set GDK_BACKEND=x11 on linux, as the conda-forge wxwidgets build
+# is compiled to use GLX, which would cause a segmentation fault
+# when running under Wayland.
+if platform.system() == 'Linux':
+    os.environ['GDK_BACKEND'] = 'x11'
+
 # Load the main version string into the package namespace
 import wx.__version__
 __version__ = wx.__version__.VERSION_STRING
@@ -20,3 +29,5 @@ from wx.core import *
 # Clean up the package namespace
 del core
 del wx
+del os
+del platform
