--- wxPython-4.0.7.post2.orig/buildtools/build_wxwidgets.py	2019-11-11 23:54:31.000000000 -0300
+++ wxPython-4.0.7.post2/buildtools/build_wxwidgets.py	2019-12-03 10:53:31.262081730 -0300
@@ -301,6 +301,7 @@
         # TODO: should there be a command line option to set the SDK?
         if sys.platform.startswith("darwin"):
             wxpy_configure_opts.append("--with-macosx-version-min=10.9")
+            wxpy_configure_opts.append("--with-macosx-sdk={}".format(os.environ.get('CONDA_BUILD_SYSROOT')))
             # for xcodePath in getXcodePaths():
             #     sdks = [ xcodePath+"/SDKs/MacOSX10.{}.sdk".format(n)
             #              for n in range(9, 15) ]
