--- wxPython-4.2.0.orig/buildtools/build_wxwidgets.py	2022-09-02 13:35:53.000000000 +0100
+++ wxPython-4.2.0/buildtools/build_wxwidgets.py	2022-09-02 13:36:33.000000000 +0100
@@ -287,7 +287,7 @@

             # Set the minimum supported OSX version.
             # TODO: Add a CLI option to set this.
-            wxpy_configure_opts.append("--with-macosx-version-min=10.10")
+            wxpy_configure_opts.append("--with-macosx-sdk={}".format(os.environ.get('CONDA_BUILD_SYSROOT')))

             # find the newest SDK available on this system
             SDK = 'none found'