diff --git a/build.py b/build.py
index dc63aed0..8dc8a388 100755
--- a/build.py
+++ b/build.py
@@ -492,11 +492,7 @@ def parseArgs(args):
     parser = makeOptionParser()
     options, args = parser.parse_args(args)
 
-    if isWindows:
-        # We always use magic on Windows
-        options.no_magic = False
-        options.use_syswx = False
-    elif options.use_syswx:
+    if options.use_syswx:
         # Turn off magic if using the system wx
         options.no_magic = True
 
