diff --git a/buildtools/build_wxwidgets.py b/buildtools/build_wxwidgets.py
index 11dbf913..037784f4 100644
--- a/buildtools/build_wxwidgets.py
+++ b/buildtools/build_wxwidgets.py
@@ -316,11 +316,11 @@ def main(wxDir, args):
             print("Universal Capable: {}".format(universalCapable))
             print("Architectures:     {}".format(arch))
 
-            wxpy_configure_opts.append("--with-libjpeg=builtin")
-            wxpy_configure_opts.append("--with-libpng=builtin")
-            wxpy_configure_opts.append("--with-libtiff=builtin")
-            wxpy_configure_opts.append("--with-libwebp=builtin")
-            wxpy_configure_opts.append("--with-regex=builtin")
+            wxpy_configure_opts.append("--with-libjpeg=sys")
+            wxpy_configure_opts.append("--with-libpng=sys")
+            wxpy_configure_opts.append("--with-libtiff=sys")
+            wxpy_configure_opts.append("--with-libwebp=sys")
+            wxpy_configure_opts.append("--with-regex=sys")
 
 
         if not options.mac_framework:
