--- wxPython-4.0.7.post2.orig/build.py	2019-11-11 23:54:31.000000000 -0300
+++ wxPython-4.0.7.post2/build.py	2019-12-03 10:51:14.355663408 -0300
@@ -1482,7 +1482,7 @@
     if not options.no_allmo:
         # Build the wx message catalogs, but first check that there is a msgfmt
         # command available
-        if findCmd('msgfmt') and findCmd('make'):
+        if findCmd('msgfmt') and not isWindows:
             locale_pwd = pushDir(posixjoin(wxDir(), 'locale'))
             print('Building message catalogs in ' + os.getcwd())
             runcmd('make allmo')
